GOVERNANCE.md

```diff
@@ -1,114 +1,160 @@
-# CRI-O GOVERNANCE.md
+# Project Governance
 
-CRI-O, being a project whose purpose is linked to fulfilling the needs of Kubernetes, has a governance structure that mirrors many SIGs and sub-projects of Kubernetes. This document highlights important aspects of, and serves as a summary of the [Kubernetes' community documentation](https://www.kubernetes.dev/docs).
+- [Values](#values)
+- [Contribution Tiers](#contribution-tiers)
+- [Becoming a Maintainer](#becoming-a-maintainer)
```

Comment:
This section is no longer present in the proposed structure of the document. I suggest removing it or explicitly adding the subsections of `Contribution Tiers`.